.github/workflows/ci.yml

```diff
@@ -7,26 +7,49 @@ on:
   - cron: '0 7 * * 6'
 jobs:
   build:
+    strategy:
+      matrix:
+        include:
+          - java: 8
+            distribution: temurin
+            sbt: 1
+          - java: 8
+            distribution: temurin
+            sbt: 2
     runs-on: ubuntu-latest
     env:
       # define Java options for both official sbt and sbt-extras
       JAVA_OPTS: -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
       JVM_OPTS:  -Xms2048M -Xmx2048M -Xss6M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v4
     - name: Setup Scala
-      uses: olafurpg/setup-scala@v10
+      uses: actions/setup-java@v4
       with:
-        java-version: "adopt@1.8"
+        distribution: "${{ matrix.distribution }}"
+        java-version: "${{ matrix.java }}"
     - name: Coursier cache
       uses: coursier/cache-action@v5
-    - name: Build and test
+    - name: Build and test (sbt 1.x)
+      if: ${{ matrix.sbt == 1 }}
+      shell: bash
       run: |
         gpg --import test-key.gpg
-        sbt -v clean test scripted
+        sbt -v ++2.12.x clean test scripted
         rm -rf "$HOME/.ivy2/local" || true
         find $HOME/Library/Caches/Coursier/v1        -name "ivydata-*.properties" -delete || true
         find $HOME/.ivy2/cache                       -name "ivydata-*.properties" -delete || true
         find $HOME/.cache/coursier/v1                -name "ivydata-*.properties" -delete || true
         find $HOME/.sbt
+    - name: Build and test (sbt 2.x)
+      if: ${{ matrix.sbt == 2 }}
+      shell: bash
+      run: |
+        gpg --import test-key.gpg
+        sbt -v ++3.x clean test scripted
+        rm -rf "$HOME/.ivy2/local" || true
+        find $HOME/Library/Caches/Coursier/v1        -name "ivydata-*.properties" -delete || true
```

Comment:
Could use a bit of insight into what this is doing. Feels a bit like they're trying to clean up specific ivy release data. Do we actually care here? Doesn't seem like anyone else is using this.

Reply:
I don't know if any of these `find .. -delete` ever mattered.

Reply:
Thanks, stripped it out.